paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -301,7 +306,7 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
             .getPMAddForm(stripeIntent, config)
         val hasUsedLink = linkStore.hasUsedLink()
 
-        val linkSignupMode = if (hasUsedLink) {
+        val linkSignupMode = if (hasUsedLink || linkSignUpDisabled) {
```

Comment:
Do we already have a test where we consume this?

Reply:
We did not test the `hasUsedLink` logic for `PaymentSheetLoader`. I added a test for that here.